data/telegraf_plugins.yml

```diff
@@ -2149,6 +2177,13 @@ output:
     introduced: 1.0.0
     tags: [linux, macos, windows, logging]
 
+  - name: GroundWork
+    id: groundwork
+    description: |
+      The  Graylog output plugin writes to a [GroundWork Monitor](https://www.gwos.com/product/groundwork-monitor/) instance.
```

Comment:
```suggestion
      The Groundwork output plugin writes to a [GroundWork Monitor](https://www.gwos.com/product/groundwork-monitor/) instance.
```

Reply:
see Scott's suggestion

Reply:
It's still not fixed though..

Reply:
@Hipska Thanks for checking - opened a PR here https://github.com/influxdata/docs-v2/pull/3803